index.src.html

```diff
@@ -2347,14 +2342,57 @@
             {{AudioData/[[resource reference]]}}.
         2. Let |reference| be a new reference to |resource|.
         3. Assign |reference| to {{AudioData/[[resource reference]]}}.
-        4. Assign the values of |data|'s {{AudioData/[[detached]]}},
+        4. Assign the values of |data|'s {{platform object/[[Detached]]}},
             {{AudioData/[[format]]}}, {{AudioData/[[sample rate]]}},
             {{AudioData/[[number of frames]]}},
             {{AudioData/[[number of channels]]}}, and
             {{AudioData/[[timestamp]]}} slots to the corresponding slots in
             |clone|.
     2. Return |clone|.
 
+: <dfn>Close AudioData</dfn> (with |data|)
+:: Run these steps:
+        1. Assign `true` to |data|'s {{platform object/[[Detached]]}} internal slot.
+        2. Assign `null` to |data|'s {{AudioData/[[resource reference]]}}.
+
+### Transfer and Serialization ###{#audiodata-transfer-serialization}
+
+NOTE: Transferring an {{AudioData}} moves its {{AudioData/[[resource
+    reference]]}} to the destination object, and "closes" the original source
+    object. Serializing an {{AudioData}} creates a "clone" of the source object,
+    resulting in two objects that refrence the same [=media resource=].
+
+: The {{AudioData}} [=transfer steps=], with |value| and |dataHolder|, are:
+:: 1. If |value|'s {{platform object/[[Detached]]}} is `true`, throw an
+        {{DataCloneError}} {{DOMException}}.
+    2. For all {{AudioData}} internal slots in |value|, assign the value of
+        each internal slot to a field in |dataHolder| with the same name as the
+        internal slot.
+    3. Run the [=Close AudioData=] algorithm with |value|.
+
+: The {{AudioData}} [=transfer-receiving steps=], with |dataHolder| and |value|, are:
+:: 1. For all named fields in |dataHolder|, assign the value of each named
+        field to the {{AudioData}} internal slot in |value| with the same name
+        as the named field.
+
+: The {{AudioData}} [=serialization steps=], with |value|, |serialized|, and  |forStorage|, are:
+:: 1. If |value|'s {{platform object/[[Detached]]}} is `true`, throw an
+        {{DataCloneError}} {{DOMException}}.
+    2. If |forStorage| is `true`, throw a {{TypeError}}.
```

Comment:
Is the right way to store PCM on disk an explicit copy of the arraybuffer, and then a recreation of an object with the same metadata, then? Is there a reason for this?

Storing decoded data is going to be fairly important for large projects in professional media creation tools.

Same for video frames.

Reply:
I hadn't appreciated that some apps might find this useful. Lets do it in a follow up. Filed https://github.com/w3c/webcodecs/issues/291